generators/typegen/generator.go

```diff
@@ -265,7 +267,23 @@ func (g *Generator) do(genConfig *config.GeneratorConfig, pkgConfig *config.Pack
 		return err
 	}
 
-	err = tmpl.Execute(f, g)
+	err = tmpl.Execute(file, g)
```

Comment:
Not a blocker, but I'm thinking we could probably write to a buffer instead of a file, and then probably format the buffer before writing it out.  That's something we can work out in another PR too, since this get's us cleaner code now.

Reply:
Yep, I was thinking the same thing. Maybe during our afternoon mob session.